src/include/basil.h

```diff
@@ -649,6 +735,13 @@ typedef struct basil_node_memory {
 	struct basil_node_memory *next;
 } basil_node_memory_t;
 
+/* This computeunit struct is added by Altair */
```

Comment:
I suggest we file a ticket to add the new pbs_basil.h at a later time and remove the reference to Altair for this change. It is OK to say something like "computeunit was added to PBS because..." but we should not be saying it was added by Altair. Your thought @lisa-altair?